server/routes.ts

```
@@ -130,13 +221,29 @@ export async function registerRoutes(app: Express): Promise<Server> {
       const documentId = parseInt(req.params.id);
       const updates = req.body;
       
-      const document = await storage.updateDocument(documentId, userId, updates);
+      // Remove fields that shouldn't be updated directly
+      delete updates.id;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)